tests/release/conftest.py

```diff
@@ -0,0 +1,158 @@
+import functools
+import os
+import time
+from collections.abc import Callable
+from pathlib import Path
+from typing import Any
+
+import pandas as pd
+import pytest
+from memory_profiler import memory_usage  # type: ignore
+
+from pseudopeople.interface import (
+    generate_american_community_survey,
+    generate_current_population_survey,
+    generate_decennial_census,
+    generate_social_security,
+    generate_taxes_1040,
+    generate_taxes_w2_and_1099,
+    generate_women_infants_and_children,
+)
+
+DATASET_GENERATION_FUNCS: dict[str, Callable[..., Any]] = {
+    "census": generate_decennial_census,
+    "acs": generate_american_community_survey,
+    "cps": generate_current_population_survey,
+    "ssa": generate_social_security,
+    "tax_w2_1099": generate_taxes_w2_and_1099,
+    "wic": generate_women_infants_and_children,
+    "tax_1040": generate_taxes_1040,
+}
+
+DEFAULT_YEAR = 2020
+DEFAULT_STATE = None
+DEFAULT_POP = "sample"
+FULL_USA_FILEPATH = "/mnt/team/simulation_science/pub/models/vivarium_census_prl_synth_pop/results/release_02_yellow/full_data/united_states_of_america/2023_08_21_16_35_27/final_results/2023_08_31_15_58_01/pseudopeople_simulated_population_usa_2_0_0"
```

Comment:
I'm not sure how best to store these filepaths. Definitely open to a constants file but would we want to go in and manually update this every time? Can we maybe use a symlink?

Reply:
I think a long time ago we had floated the idea with RT about using versions of data so we can valdiate if a data version and package version are incompatible and we could probably store paths with that info but this seems fine for now.

Reply:
Yeah, like Jim said we've discussed where to store this stuff a few times. I agree this seems fine for now. We _do_ have symlinks that point to this but I'm _always_ confused with how they work (b/c we have state-specific as well as full and also iirc they're not really be generated at the most intuitive location).